plugin/src/theme.ts

```diff
@@ -72,8 +72,12 @@ export const theme = {
     sm: DEFAULT_THEME.shadows.sm(rgba(localTheme('colors.kale.600'), 0.15)),
     DEFAULT: DEFAULT_THEME.shadows.md(rgba(localTheme('colors.kale.600'), 0.15)),
     lg: DEFAULT_THEME.shadows.lg('20px', '28px', rgba(localTheme('colors.kale.600'), 0.15)),
-    inner: `inset ${DEFAULT_THEME.shadows.md(rgba(localTheme('colors.blue.600'), 0.35))}`,
-    outline: DEFAULT_THEME.shadows.md(rgba(localTheme('colors.blue.600'), 0.35)),
+    inner: `inset ${DEFAULT_THEME.shadows.xs(
+      localTheme('colors.white')
+    )}, inset ${DEFAULT_THEME.shadows.md(localTheme('colors.blue.600'))}`,
+    outline: `${DEFAULT_THEME.shadows.xs(localTheme('colors.white'))}, ${DEFAULT_THEME.shadows.md(
```

Comment:
Yeah I completely agree and thought the same when I encountered this. For sure, we can use `DEFAULT_THEME.colors.background`; for a more configurable experience, any opposition to `localTheme('colors.background')` and adding it to the theme `colors` object?

Reply:
I think we left `background` and other keys (`primaryHue`, `successHue`, etc) out of the theme because they don't really jive with the Tailwind [color object syntax](https://tailwindcss.com/docs/customizing-colors#color-object-syntax).